sklearn/feature_selection/tests/test_rfe.py

```diff
@@ -237,7 +261,13 @@ def test_rfecv_mockclassifier():
     rfecv = RFECV(estimator=MockClassifier(), step=1)
     rfecv.fit(X, y)
     # non-regression test for missing worst feature:
+
+    # TODO: Remove in v1.2 when grid_scores_ is removed
     assert len(rfecv.grid_scores_) == X.shape[1]
```

Comment:
I would rather use `with pytest.warns(FutureWarning, match=msg):` locally instead of using `@ignore_warnings(category=FutureWarning)` for the whole test.